### DISCUSSION TITLE
Control date format on footer from git

### DISCUSSION THREAD

**Discussion by MrKevHunter** [vuepress-theme-hope/vuepress-theme-hope#1726] [Q&A] [answered]
I want the date format in the footer from git to appear in UK (DD/MMM/YYYY) format, 

https://vuepress-theme-hope.github.io/v2/guide/feature/meta.html#git-based-information

> The theme will use dayjs to automatically localize the text of last update time according to the current language.

Is this the current language specified in the config - if so do I need to submit a PR for a en-gb lanaguage?

**Comment by Mister-Hope** [comment, marked as answer]
Sorry, we no longer use dayjs to format now in v2, (we will correct v2 docs soon).

Now we are using `Date.toLocaleString(currentPageLang)`

So if the browser is supporting your language, it should provide the correct locale string.

We do not have plans to make any edit here, but you are free to overide it using https://vuepress-theme-hope.github.io/v2/cookbook/advanced/replace.html#replace-components

**Reply by MrKevHunter** [reply]
I was assuming that if I added 

```
  locales: {
    '/': { 
      lang: 'en-GB',
    }
  },
```

 to `defineHopeConfig` now that it would do the date, but what actually happens is the page doesn't render at all and I see

`en-GB has no path config, and will return '/' instead.` in the console.

Have i done something wrong?

**Reply by Mister-Hope** [reply]
That 's just a warning, ignore it.

**Reply by Mister-Hope** [reply]
Improved logging in https://github.com/vuepress-theme-hope/vuepress-theme-hope/commit/de726a004c2d9f145a7f0e6952e2888f55eeaa2a

**Reply by MrKevHunter** [reply]
If I do that I get no content when using 

`npm run docs:dev`

**Reply by Mister-Hope** [reply]
Just wait for a new version.

**Reply by MrKevHunter** [reply]
Thanks @Mister-Hope
